Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -1006,6 +1006,64 @@ final class CodeActionTests: XCTestCase {
     }
   }
 
+  func testConvertStringConcatenationToStringInterpolation() async throws {
+    try await assertCodeActions(
+      """
+      1️⃣#"["# + 2️⃣key + ": \\(3️⃣d) " + 4️⃣value + ##"]"##5️⃣
+      """,
```

Comment:
If you use a raw multiline string literal (with `#"""`), you shouldn’t need to escape the `\`, which should make the test easier to reason about. Same in the expected new text.